Assets/01_Scripts/GameInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameInfo : MonoBehaviour
```

Comment:
I will appreciate that you change the name class for something more meaningful, is not clear about what kind of 'info' we are talking about.